app/controllers/api/v1/statuses_controller.rb

```diff
@@ -77,9 +77,9 @@ def reblog
   end
 
   def unreblog
-    reblog         = Status.where(account_id: current_user.account, reblog_of_id: params[:id]).first!
-    @status        = reblog.reblog
-    @reblogged_map = { @status.id => false }
+    reblog       = Status.where(account_id: current_user.account, reblog_of_id: params[:id]).first!
```

Comment:
You can also do `Status.find_by!(account: current_user.account, reblog_of_id: params[:id])`